test/e2e_authorized_test.go

```diff
@@ -0,0 +1,69 @@
+package test
+
+import (
+	"net/http"
+	"testing"
+	"time"
+)
+
+func TestAuthorized(t *testing.T) {
+
+	users := []User{
+		{
+			Username: "foo",
+			Password: "bar",
+		},
+		{
+			Username: "lobby",
+			Password: "niu",
+		},
+	}
+	go startEnvoy(users)
```

Comment:
Why do we need to use the "go" keyword when executing the command "envoy -c envoy.yaml" is already running in the background?
```go
err = exec.Command("bash", "-c", "envoy -c envoy.yaml &").Run()
```

Reply:
You are Right, it doesn't need to use goroutine here. I will fix it.